installation/README.md

```diff
@@ -0,0 +1,25 @@
+# Compass installation and testing
+
+## Overview
+
+This document presents the deep dive into Compass installation and testing.
+
+## Installation
```

Comment:
Add the following paragraph below the header:

```

 To run the Compass with Kyma locally, run the following script:

 ```bash
./installation/scripts/testing.sh
```

```